### PR TITLE
Fix sequence init

### PR DESCRIPTION
**Pull request by Flova**
# Summary
See #107 

## Proposed changes
- Move action init into sequence class 
- Remove unused cmake file

### CODE REVIEW

File: dynamic_stack_decider/dynamic_stack_decider/sequence_element.py [hunk 1]
```diff
@@ -16,21 +17,44 @@ class SequenceElement(AbstractStackElement):
...
+        self,
+        blackboard,
+        dsd: "DSD",
+        actions: list[ActionTreeElement],
+        init_function: Callable[[AbstractTreeElement], AbstractStackElement],
+    ):
         """
+        :param blackboard: Shared blackboard for data exchange and code reuse between elements
+        :param dsd: The stack decider which has this element on its stack.
         :param actions: list of initialized action elements
```
Comment (jaagut) [inline dynamic_stack_decider/dynamic_stack_decider/sequence_element.py]: I guess they are NOT initialized?

File: dynamic_stack_decider/dynamic_stack_decider/sequence_element.py [hunk 2]
```diff
@@ -16,21 +17,44 @@ class SequenceElement(AbstractStackElement):
...
 
     def perform(self, reevaluate=False):
         """
         Perform the current action of the sequence. See AbstractStackElement.perform() for more information
 
         :param reevaluate: Ignored for SequenceElements
         """
+        # Log the active element
+        self.publish_debug_data("Active Element", self.current_action.name)
+        # Pass the perform call to the current action
```
Comment (jaagut) [inline dynamic_stack_decider/dynamic_stack_decider/sequence_element.py:53]:
```suggestion
        # Call perform on the current action
```

Comment (Flova) [reply dynamic_stack_decider/dynamic_stack_decider/sequence_element.py:53]: No we implement a perform call and pass that on to the currently active action.

Comment (Flova) [reply dynamic_stack_decider/dynamic_stack_decider/sequence_element.py:53]: The other thing is obvious imo.

Comment (jaagut) [reply dynamic_stack_decider/dynamic_stack_decider/sequence_element.py:53]: I understand passing as the transfer of some data from some A to B. But you don't transfer anything around, you just call perform on B

**Review by jaagut — APPROVED** [review]

**Pull request merged**